english/hulk/wake-up/whats-next.md

```diff
@@ -0,0 +1,3 @@
+Sitting up on the floor with a familiar confused drowsiness, you look around through Bruce Banner's eyes at the scene around you.  "Turtles?  Why are there turtles everywhere?  And why are they all starring at me?", you mumble.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.